src/calculateRentalCost.js

```diff
@@ -28,6 +28,15 @@
  */
 
 function calculateRentalCost(days) {
+  const price = days * 40;
+
+  if (days < 3) {
+    return price;
+  } else if (days < 7) {
+    return price - 20;
+  } else {
+    return price - 50;
+  }
```

Comment:
You can remove last else:
```suggestion
  return price - 50;
```